tests/acceptance/test_multi_threading.py

```diff
@@ -187,7 +187,7 @@ def test___shared_interpreter___run_multiple_acquisitions_with_events___callback
             )
             for i, device in enumerate(multi_threading_test_devices)
         ]
-        samples_per_chan = [1000 for _ in tasks]
```

Comment:
either split this into 2 PRs or update the title to be slightly more generic about what you're doing. I prefer splitting them, personally.